mandatory/3-playing-computer.js

```diff
@@ -8,11 +8,11 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? Awnser: 2,6,4,9,6,13,8
+  4. How many times is "f1" called?  Awnser: 2 times (2 odd number 1 3)
+  5. How many times is "f2" called?  Awnser: 3 times (3 even number 0 2 4)
+  6. What value does the "a" parameter take in the first "f1" call?  Awnser: 8
```

Comment:
Declaration of `f1`:

`
const f1 = function (a, b) {
  return a + b;
};
`

We can see from the creation of f1 that its first parameter is a. Therefore whenever f1 is called, its first argument will be assigned to the first parameter a. Let's look at where f1 is called:

`
} else {
  const e = f1(i, a);
  console.log(e);
}
`
`f1` is called when `i` is either 1 or 3.

Therefore, the first time `f1` is called `i` must be equal to 1.

So `f1`'s first argument is 1 and therefore inside the function `f1`, `a` will be assigned a value of 1